posthog/api/session_recording.py

```diff
@@ -201,3 +214,18 @@ def snapshots(self, request: request.Request, **kwargs):
                 }
             }
         )
+
+    # Returns metadata given a list of session recording ids
+    @action(methods=["GET"], detail=False)
+    def metadata(self, request: request.Request, **kwargs):
+        filter = SessionRecordingsFilter(request=request)
+        session_recordings_metadata = self._get_session_recording_list_meta_data(filter)
+
+        if not request.user.is_authenticated:  # for mypy
+            raise exceptions.NotAuthenticated()
+
+        session_recording_serializer = SessionRecordingMetaDataSerializer(data=session_recordings_metadata, many=True)
+
+        session_recording_serializer.is_valid(raise_exception=True)
+
+        return Response({"results": session_recording_serializer.data})
```

Comment:
I think we should rename this to something like `properties` so as not to get confused with the other `metadata`  and we can probably minimise all the implementation code to one simple function.